mapp/0.0.9/Dockerfile

```diff
@@ -0,0 +1,52 @@
+###############################################################################
+#
+#   Dockerfile for the container
+#
+#   AUTHOR: Maciej_Bak
+#   AFFILIATION: Swiss_Institute_of_Bioinformatics
+#   CONTACT: maciej.bak@unibas.ch
+#   CREATED: 06-10-2019
+#   LICENSE: GPL
+#
+###############################################################################
+
+# BASE IMAGE
+FROM ubuntu:18.04
```

Comment:
You could use [Miniconda](https://hub.docker.com/r/continuumio/miniconda/) as your base image to considerably reduce build time...

Reply:
@AngryMaciek Can you please try this?

Reply:
Yes, I will try and let you know :)